autoprecompiles/src/basic_blocks.rs

```diff
@@ -0,0 +1,112 @@
+use std::collections::BTreeSet;
+
+use serde::{Deserialize, Serialize};
+
+use crate::SymbolicInstructionStatement;
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct SymbolicBlock<T> {
+    pub start_idx: usize,
```

Comment:
Please document - just to be sure - this is the instruction index, not the byte-offset inside the program blob, right?

Reply:
correct, done